AirCasting/Dashboard/Reordering/ReorderingDashboard.swift

```diff
@@ -20,13 +20,19 @@ struct ReorderingDashboard: View {
             LazyVGrid(columns: columns) {
                 ForEach(viewModel.sessions, id: \.uuid) { session in
                     ReoredringSessionCard(session: session, thresholds: viewModel.thresholds)
-                        .overlay(viewModel.currentlyDraggedSession?.uuid == session.uuid && changedView ? Color.white.opacity(0.8) : Color.clear)
+                        .overlay(viewModel.currentlyDraggedSession?.uuid == session.uuid && changedView ? Color.aircastingWhite.opacity(0.8) : Color.clear)
                         .onDrag({
                             viewModel.currentlyDraggedSession = session
                             changedView = false
                             return NSItemProvider(object: String(describing: session.uuid) as NSString)
                         })
                         .onDrop(of: [.text], delegate: DropViewDelegate(sessionAtDropDestination: session, currentlyDraggedSession: $viewModel.currentlyDraggedSession, sessions: $viewModel.sessions, changedView: $changedView))
+                        .trailingSwipeAction {
+                            withAnimation(.easeOut(duration: 0.5)) {
+                                viewModel.sessions.removeAll(where: { $0.uuid == session.uuid })
```

Comment:
let's not operate on a viewmodels properties directly like that. The correct way will be to create a special purpose VM method for that